pkg/api/staticHandlers.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// StaticHandlers.go exposes an implementation of MethodHandlers with static (compile time) configs.
+
+package api
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/struct"
+
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"google.golang.org/genproto/googleapis/rpc/status"
+
+	denyadapter "istio.io/mixer/adapter/denyChecker"
```

Comment:
thinking out loud: do we need StaticHandlers to have denyChecker hardcoded in? What if we changed `NewStaticHandlers()` to be something like:
```
type StaticBinding struct {
    RegisterFn func(r registry.Registrar) error
    Manager aspectsupport.Manager
    // related CombinedConfig stuff here too, I guess.
}

func NewStaticHandlers([]StaticBinding) *StaticHandlers {
...
    if err := binding.RegisterFn(registry); err != nil {
...
    managers.append(managers, binding.Manager)
...
```

Then we could have //cmd/server depend on //adapters for the time being and avoid having //pkg/api depend on //adapters (maybe not all that important, but feels useful).

It might even allow us to build a noconfig_server.go which we run until the config-full version is ready.

Thoughts? Things I've missed?